### PR TITLE
Convert PullRequestStatus to a union of types

### PR DESCRIPTION
**Pull request by samuelli**

### CODE REVIEW

File: src/types/api.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  reviewers: string[];
+}
+
+// Actionable - outgoing
+
+// Viewer's pull PR requires changes
+interface PendingChanges {
+  type: 'PendingChanges';
+}
+// Merge required by viewer
```
Comment (aomarks) [inline src/types/api.ts]: add some newlines between these interfaces?

File: src/types/api.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+export interface Review {
+  author: string;
+  createdAt: number;
+  reviewState: PullRequestReviewState;
+}
+
+interface UnknownStatus {
+  type: 'UnknownStatus';
+}
+// Not necessarily actionable.
```
Comment (aomarks) [inline src/types/api.ts]: move comment up

File: src/types/api.ts (hunk 1 above)
Comment (samuelli) [reply src/types/api.ts]: Done

File: src/types/api.ts (hunk 2 above)
Comment (samuelli) [reply src/types/api.ts]: Done.

**Pull request merged**